### DISCUSSION TITLE
Can you get a list of users that liked or retweeted a tweet?

### DISCUSSION THREAD

**Discussion by lukeshepard21** [Xwilarg/TwitterSharp#42] [Q&A] [answered]
Can you get a list of users that liked or retweeted a tweet? Can either be userIds or twitter handles

**Comment by marcogruhl** [comment, marked as answer]
It is not implemented yet.
[get-tweets-id-liking_users](https://developer.twitter.com/en/docs/twitter-api/tweets/likes/api-reference/get-tweets-id-liking_users)
[get-tweets-id-retweeted_by](https://developer.twitter.com/en/docs/twitter-api/tweets/retweets/api-reference/get-tweets-id-retweeted_by)

Maybe i will find some time to implement it soon.

**Reply by marcogruhl** [reply]
done #43

**Reply by lukeshepard21** [reply]
cheers mate. appreciate it
